Makefile.envs

```diff
@@ -149,6 +149,7 @@ export MAIN_MODULE_LDFLAGS= $(LDFLAGS_BASE) \
 	-legl.js \
 	-lwebgl.js \
 	-lhtml5_webgl.js \
+	-lsdl.js \
```

Comment:
The change in file size (pyodide.asm.js) due to this addition is as follows:

__main branch__

```
    Original size: 1112 KB
    Gzip compressed size (level 1): 248 KB
    Gzip compressed size (level 6): 203 KB
    Gzip compressed size (level 9): 203 KB
    Brotli compressed size: 170 KB
```

__This PR__

```
    Original size: 1194 KB
    Gzip compressed size (level 1): 272 KB
    Gzip compressed size (level 6): 223 KB
    Gzip compressed size (level 9): 223 KB
    Brotli compressed size: 187 KB
```